src/scaladoc/scala/tools/nsc/doc/html/HtmlFactory.scala

```diff
@@ -96,6 +95,10 @@ class HtmlFactory(val universe: doc.Universe, val reporter: Reporter) {
     "ownerbg2.gif"
   )
 
+  final def webjarResources = List(
+    ("jquery.min.js", "CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=")
```

Comment:
a comment saying what version of jQuery has this SRI hash wouldn't be a bad idea